src/main/java/com/iexec/common/utils/SignatureUtils.java

```diff
@@ -39,6 +40,27 @@ public static boolean doesSignatureMatchesAddress(byte[] signatureR,
         return false;
     }
 
+    public static boolean isSignatureValid(String message, Signature sign, String signerAddress) {
+        return isSignatureValid(message.getBytes(), sign, signerAddress);
+    }
+
+    public static boolean isSignatureValid(byte[] message, Signature sign, String signerAddress) {
```

Comment:
This is an important method here, good unit tests are mandatory.

Reply:
Sure yes! thank you for the note.